src/lmnt/api.py

```diff
@@ -306,6 +310,8 @@ async def synthesize(self, text: str, voice: str, **kwargs):
     length = kwargs.get('length', None)
     if length is not None:
       form_data.add_field('length', length)
+    if 'quality' in kwargs:
```

Comment:
This isn't actually documented (I think that's ok, not sure we've documented it anywhere yet, just mentioning while reading).